tests/test_fitsChan.py

```diff
@@ -84,6 +84,47 @@ def insertPixelMapping(self, mapping, frameSet):
         frameSet.addFrame(ast.FrameSet.CURRENT, iwcToSky, skyFrame)
         return frameSet
 
+    def test_FitsChanAttributes(self):
```

Comment:
I think it would be good to have a docstring here saying that this tests the default values and attributes (which function like python properties, I think?).